modules/sdk-core/src/bitgo/bitgoBase.ts

```diff
@@ -0,0 +1,43 @@
+import * as superagent from 'superagent';
+import { EnvironmentName } from './environments';
+
+export interface BitGoRequest<ResultType = any> extends superagent.SuperAgentRequest {
```

Comment:
most of these interfaces already exist in `@bitgo/sdk-api`. Those should be moved here, not duplicated.

Reply:
Do you mean we should create a folder called `sdk-api` in `sdk-core` and create a file i.e `api.ts` to define the BitGoRequest interface?

Reply:
yes